### PR TITLE
Meet the team e2e test

### PR DESCRIPTION
**Pull request by boodland**
### Issue link / number:
#1089

### What changes did you make?
Created meet-the-team e2e test for english version

### Why did you make the changes?
So the page is covered with test and help us to validate it is working when making changes or migrating it to app router

### Did you run tests?
Yes

### CODE REVIEW

**Comment by boodland** [comment]
@kyleecodes a follow up issue/ticket can be created to add tests for the rest of languages and complete the fixture with the rest of the supporting team members, I did not include all of them to save time and be able to move to another migration task. Thanks in advance

File: components/storyblok/StoryblokTeamMemberCard.tsx [hunk 1]
```diff
@@ -127,7 +128,7 @@ const StoryblokTeamMemberCard = (props: StoryblokTeamMemberCardProps) => {
       </CardActionArea>
       <Collapse in={expanded} timeout="auto" unmountOnExit>
         <CardContent sx={collapseContentStyle}>
-          <Typography variant="body2" mb={0} paragraph>
+          <Typography variant="body2" mb={0} paragraph component={'div'}>
```
Comment (eleanorreem) [inline components/storyblok/StoryblokTeamMemberCard.tsx:131]: @boodland - this was originally set as a paragraph component. Is there a reason why you wanted to set it to div?

File: cypress/fixtures/meet-the-team.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (eleanorreem) [inline cypress/fixtures/meet-the-team.json:1]: @boodland is this file used?

**Comment by eleanorreem** [comment]
@boodland thanks for these tests. Super thorough - one small question on reassigning a typography component to a div!

File: components/storyblok/StoryblokTeamMemberCard.tsx (hunk 1 above)
Comment (boodland) [reply components/storyblok/StoryblokTeamMemberCard.tsx:131]: Hi @eleanorreem, semantically it is not allowed to have a p tag inside another p tag. React hydratation is throwing an error due to that, I have set to a div as the text content is in the deeper p tag

Comment (eleanorreem) [reply components/storyblok/StoryblokTeamMemberCard.tsx:131]: That makes total sense now. I see what the issue is - richText is being rendered inside a paragraph. Not ideal! I'd just remove the "paragraph" prop as well.

**Pull request merged**